src/content/docs/journey-migration/migrating-infra.mdx

```diff
@@ -0,0 +1,230 @@
+
+<img
+    title="infrastructure_dashboards"
+    alt="An image displaying New Relic's infrastructure monitoring dashboard"
+    src={journeyMainDash}
+/>
+
+## Your infrastructure monitoring solution [#infra-solution]
+
+New Relic makes it easy for you to observe your entire infrastructure, whether it runs in the cloud or dedicated hosts. In Datadog, you may be familiar with these capabilities as `agents`, and we call the same infrastructure capabilities `agents` at New Relic as well. Here are a few of the offerings for New Relic infrastructure monitoring:
+
+* Infrastructure agents for Linux, macOS, and Windows operating systems with guided and manual install options.
+* An agent for for [Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/) that can be set up in a number of ways.
+* Cloud agents for Amazon, Azure, and Google Cloud Platform that you can install independently of our infrastructure agents.
+* On-host integration agents for services like MySQL, Apache, Docker, and Kafka that work together with our infrastructure agent to maximize your data reporting.
+
+As long as your system and on-host integrations meet our [minimum requirements](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent/), all you need to try is a [free New Relic account](https://newrelic.com/signup) and you can start right away.
+
+## Get started with New Relic infrasctructure monitoring [#get-started]
+
+Your infrastructure data in New Relic is located in the [Infrastructure](https://one.newrelic.com/infra) section of the UI, exactly the same as Datadog. The fastest way to get an infrastructure monitor up and running can be done through our [guided installation](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGl2ZUNvbXBvbmVudCI6IlZUU09FbnZpcm9ubWVudCIsInBhdGgiOiJndWlkZWQifQ==) path, which you can find within [New Relic](https://one.newrelic.com) by selecting **Add Data > All > Guided install**.
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Windows
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Linux
+        </TabsBarItem>
+        <TabsBarItem id="3">
+            macOS
+        </TabsBarItem>
+        <TabsBarItem id="4">
+            Kubernetes
+        </TabsBarItem>
+        <TabsBarItem id="5">
+            Docker
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+        <SideBySide>
+        <Side>
+            <img
+            title="Windows guided install"
+            alt="An image displaying New Relic's guided installation for Windows"
+            src={journeyGuidedWindows}
+            />
+        </Side>
+        <Side>
+        <Steps>
+            <Step>
```

Comment:
Suggest moving the UI pathway/link from the ¶ above these steps into the first step in the flow. Thaat avoids reader bouncing back and forth to see where they need to go.

Reply:
A small counterpoint: the install process doesn't begin until after the guided install option is selected, so having the path be above each individual step reduces a step, making the process seem shorter. The procedure also leads them up through that path before breaking off instructions into the various sub-processes for installing each agent.